apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -784,6 +786,22 @@ export const renderError = ({
     if (tmpError.message === "User refused") {
       tmpError = new TransactionRefusedOnDevice();
     }
+  } else if(tmpError instanceof NoSuchAppOnProvider) {
+    return (
+      <NoSuchAppOnProviderErrorComponent
+        Icon={
```

Comment:
not needed if we add it directly inside the component or you can set the new design from here

Reply:
it was already used like this in the last return of this function , that's why i just copied it. 
i've edit it the way you suggest. thanks